reviews/2022-open-cluster-management-annual.md

```diff
@@ -0,0 +1,98 @@
+
+Open Cluster Management (OCM) provides a framework to enable any capability within the Kubernetes ecosystem to orchestrate itself across multiple clusters and cloud providers. The built-in primitives allow an understanding of cluster inventory, cluster workload placement, and cluster workload distribution. The hub-agent model of OCM allows for new kinds of agent behavior to be injected into managed clusters easily. New behaviors can further configure clusters or collect information about clusters (such as observability data or search indexing). The project also provides two out of the box capabilities: multicluster application delivery and orchestration, multicluster policy-based governance. Additional integrations and add-ons can be easily developed through the use of OCM robust Add-on Framework.
+
+
+### Alignment with CNCF
+
+- OCM was accepted as a CNCF Sandbox project on November 09, 2021.
+
+## Development metrics
+
+The OCM devstats page and dashboards can be found [here](https://openclustermanagement.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m&search=open).
+
+- According to devstats, OCM currently has [64](https://openclustermanagement.devstats.cncf.io/d/22/prs-authors-table?orgId=1) contributors from [19](https://openclustermanagement.devstats.cncf.io/d/5/companies-table?orgId=1)
+companies. On average, there are [~1085 contributions per month](https://openclustermanagement.devstats.cncf.io/d/74/contributions-chart?orgId=1&var-period=m&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-company_name=All&var-company=all&from=now-1y&to=now) contained within [134 merged PRs per month](https://openclustermanagement.devstats.cncf.io/d/74/contributions-chart?orgId=1&var-period=m&var-metric=prs&var-repogroup_name=All&var-country_name=All&var-company_name=All&var-company=all&from=now-1y&to=now).
+- [New PRs in last year](https://openclustermanagement.devstats.cncf.io/d/15/new-prs-in-repository-groups?orgId=1).
+- The community has grown since the project entered the CNCF sandbox.
+  - We held weekly community meetings constantly. The meeting records can be found in [here](https://www.youtube.com/@OpenClusterManagement/videos) for different timezone/language community members. The average number of meeting attendees is ~10.
+  - Number of contributors: 120+ -> **150+**
+  - Github stars: 600+ -> **1000+**
+  - Github forks: 330+ -> **560+**
+  - Contributing organizations: 5 -> **19**
+
+Regarding these metrics, we would perceive the community as healthy and rising.
+
+## Maintainers
+
+We have established [the contributor ladder](https://github.com/open-cluster-management-io/community/blob/main/CONTRIBUTOR_LADDER.md), and quite a few active and qualified contributors joined to become members. We currently have [7 maintainers from 3 organizations](https://github.com/cncf/foundation/blob/main/project-maintainers.csv) as maintainers.
```

Comment:
Hi @qiujian16 for the `CONTRIBUTOR_LADDER` it will nice if we can merge this PR: `https://github.com/open-cluster-management-io/community/pull/72`